osbs-box.py

```diff
@@ -239,6 +239,15 @@ def _pull_image(source, destination_repo):
     _run(['docker', 'push', destination])
 
 
+def _copy_image(source, destination_repo):
+    destination = 'docker://{}/{}'.format(LOCAL_REGISTRY, destination_repo)
+    if not source.startswith('docker://'):
+        source = 'docker://' + source
+    print('Copying {} to {}'.format(source, destination))
+    _run(['docker', 'run', '--rm', '{}_bodhi-skopeo-lite'.format(dir_path), 'copy',
```

Comment:
What about disabling tls verify for source? Some of our registries need that unfortunately.

Reply:
updated